tests/nocover/test_coverage.py

```diff
@@ -85,7 +88,11 @@ def test(a, b, c):
         branch=branch, timid=timid,
     )
     cov._warn = escalate_warning
-    cov.start()
+    with warnings.catch_warnings():
+        # Workaround, until https://github.com/nedbat/coveragepy/pull/33
```

Comment:
References pull is closed and seems master has a fix. Workaround is still actual?

Reply:
Ooh, the release was yesterday!  In a few hours (after @pyup-bot bumps our dependencies), I'll take out this commit.